src/go/rdctl/cmd/root.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright © 2022 SUSE LLC
+
+*/
+
+// Package cmd is the main package for this CLI
+package cmd
+
+import (
+  "encoding/json"
+  "fmt"
+  "io/ioutil"
+  "log"
+  "net/http"
+  "os"
+  "path/filepath"
+  "strconv"
+
+  "github.com/spf13/cobra"
+)
+
+var (
+  // Used for flags
+  configPath  string
+  user string
+  port string
+  password string
```

Comment:
Not really a fan of the global variables here (because tracking down where the magic variables come from within functions is a _tiny_ bit of thought).  Not sure what my suggestion is; maybe a struct so that there's only one variable that is global (though it holds the same number of state)?

Especially since this is mostly `CLIConfig`…

Reply:
They're unexported module-level globals, and I'm expecting that we'll have more files, so I don't see a big problem here.

Also there are now 7 vars, not 4, that are also used for selecting the config file. So no changes deemed necessary now.